metaflow/plugins/aws/step_functions/step_functions_cli.py

```diff
@@ -78,9 +78,15 @@ def step_functions(obj):
               help="Add a decorator to all steps. You can specify this "
                    "option multiple times to attach multiple decorators "
                    "in steps.")
+@click.option('--step-function-name',
```

Comment:
We can rename it to `--name` to keep it simple. Do you have any thoughts around how `step-functions trigger` command would work in this scenario, where the user overrides the name of the workflow?